### PR TITLE
Minor text changes and small corrections

### PR DESCRIPTION
**Pull request by demobox**
Using this for review only; will create an appropriate branch in the backing repo so this isn't removed

### CODE REVIEW

File: _posts/2018-01-11-keystone-v3.md [hunk 1]
```diff
@@ -58,23 +58,27 @@ KeystoneApi keystone = ContextBuilder.newBuilder("openstack-keystone-3")
...
 
-* Class `KeystoneProperties` has been moved to package `org.jclouds.openstack.keystone.config`.
-* Class `CredentialTypes` has been moved to package `org.jclouds.openstack.keystone.auth.config`.
+* Class `KeystoneProperties` has been moved to `org.jclouds.openstack.keystone.config`.
+* Class `CredentialTypes` has been moved to `org.jclouds.openstack.keystone.auth.config`.
 * The `KeystoneAuthenticationModule` and the `AuthenticationApiModule` have been refactored and generalised into:
   * `AuthenticationModule` - Providing authentication services to all OpenStack APIs and providers.
   * `ServiceCatalogModule` - Providing endpoint resolution to all OpenStack APIs and providers.
+
+Replace uses of `KeystoneAuthenticationModule` with `AuthenticationModule`, and uses of `AuthenticationApiModule` with `ServiceCatalogModule`.
```
Comment (demobox) [inline _posts/2018-01-11-keystone-v3.md]: @nacx Is this advice correct?

File: _posts/2018-01-11-keystone-v3.md [hunk 2]
```diff
@@ -25,27 +25,27 @@ overrides.put(KeystoneProperties.KEYSTONE_VERSION, "3");
...
+// Project scoped authorization (can use the project name or the ID)
 overrides.put(KeystoneProperties.SCOPE, "project:jclouds");
 overrides.put(KeystoneProperties.SCOPE, "projectId:2f9b30f706bc45d7923e055567be2e98");
 // Domain scoped authorization (can use the domain name or the ID)
 overrides.put(KeystoneProperties.SCOPE, "domain:default");
 overrides.put(KeystoneProperties.SCOPE, "domainId:2f9b30f706bc45d7923e055567be2e98");
 {% endhighlight %}
 
-# Using the Keystone V3 APIs
+Credentials in Keystone V3 must include the `domain` name and the `username`, as shown above.
```
Comment (demobox) [inline _posts/2018-01-11-keystone-v3.md]: Not sure what this exactly is referring to - where is `domain` and `username` in the examples?

**Comment by jclouds-commentator** [comment]
Go to http://a941b2b3734aeff9f674-ce5348461b057942a1e66bac883e092b.r35.cf5.rackcdn.com/ to review your changes.

**Comment by demobox** [comment]
Reopened as #207

**Pull request closed without merging**